packages/sources/dlc-btc-por/src/transport/proof-of-reserves.ts

```diff
@@ -155,23 +153,23 @@ export class DLCBTCPorTransport extends SubscriptionTransport<TransportTypes> {
   }
 
   async verifyVaultDeposit(vault: RawVault, attestorPublicKey: Buffer) {
-    if (!vault.fundingTxId || !vault.taprootPubKey || !vault.valueLocked || !vault.uuid) {
+    if (!vault.taprootPubKey || !vault.valueLocked || !vault.uuid) {
       return false
     }
+    const txID = vault.wdTxId ? vault.wdTxId : vault.fundingTxId
```

Comment:
If you wanted to be really fancy:
```suggestion
    const txID = vault.wdTxId ?? vault.fundingTxId
```